codenames-domain/src/main/java/com/codenames/services/GameService.java

```diff
@@ -1,60 +1,68 @@
 package com.codenames.services;
 
-import com.codenames.dto.RoomDto;
 import com.codenames.exceptions.RoomLimitIsOver;
 import com.codenames.models.game.CodeNamesGame;
 import com.codenames.models.game.User;
 import com.codenames.models.room.Room;
 import com.codenames.models.game.Player;
-import com.codenames.properties.CodeNamesGameProperties;
-import com.codenames.properties.DefaultMessagePathProperties;
+import com.codenames.properties.CodeNamesProperties;
+import com.codenames.properties.WSResponsePathProperties;
 import com.jjerome.dto.Request;
 import com.jjerome.models.MessageSender;
-import jakarta.annotation.PostConstruct;
-import lombok.RequiredArgsConstructor;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Service;
 
 import java.util.Random;
 
 @Service
-//@RequiredArgsConstructor
 public class GameService {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(GameService.class);
 
     private final Random random = new Random();
 
-    private final CodeNamesGameProperties gameProperties;
-
     private final MessageSender messageSender;
 
     private final PlayerService playerService;
 
     private final RoomService roomService;
 
-    private final DefaultMessagePathProperties defaultMessagePathProperties;
+    private final WordsService wordsService;
+
+    private final CodeNamesProperties codeNamesProperties;
+
+    private final WSResponsePathProperties wsResponsePathProperties;
 
     private final int minRoomID;
 
     private final int maxRoomID;
 
     private final int roomLimit;
 
-    public GameService(CodeNamesGameProperties gameProperties,
-                       MessageSender messageSender,
+    public GameService(MessageSender messageSender,
                        PlayerService playerService,
                        RoomService roomService,
-                       DefaultMessagePathProperties defaultMessagePathProperties) {
-        this.gameProperties = gameProperties;
+                       WordsService wordsService,
+                       CodeNamesProperties codeNamesProperties) {
         this.messageSender = messageSender;
         this.playerService = playerService;
         this.roomService = roomService;
-        this.defaultMessagePathProperties = defaultMessagePathProperties;
-        this.minRoomID = gameProperties.getMinRoomId();
-        this.maxRoomID = gameProperties.getMaxRoomId();
+        this.wordsService = wordsService;
+        this.codeNamesProperties = codeNamesProperties;
+        this.wsResponsePathProperties = codeNamesProperties.getWsResponsePaths();
+        this.minRoomID = codeNamesProperties.getGameRooms().getMinRoomId();
+        this.maxRoomID = codeNamesProperties.getGameRooms().getMaxRoomId();
         this.roomLimit = maxRoomID - minRoomID;
+//        System.out.println(codeNamesProperties);
```

Comment:
Remove

Reply:
Sorry, the old version of the file has already been deleted